src/main/java/com/danielagapov/spawn/Mappers/ChatMessageMapper.java

```diff
@@ -35,9 +37,19 @@ public static List<ChatMessageDTO> toDTOList(List<ChatMessage> chatMessages) {
                 .collect(Collectors.toList());
     }
 
-    public static List<ChatMessage> toEntityList(List<ChatMessageDTO> chatMessageDTOs) {
+    public static List<ChatMessage> toEntityList(List<ChatMessageDTO> chatMessageDTOs, List<User> users, List<Event> events) {
         return chatMessageDTOs.stream()
-                .map(ChatMessageMapper::toEntity)
+                .map(dto -> {
```

Comment:
Same as with the individual `toEntity` method, these lists of users and events aren't meant to be in this file, but at the service layer. So, for this method, please change it to, and let's move this logic.

public static List<ChatMessage> toEntityList(List<ChatMessageDTO> chatMessageDTOs) {
    return chatMessageDTOs.stream()
            .map(ChatMessageMapper::toEntity)
            .collect(Collectors.toList());
}